test/__snapshots__/integration.test.js.snap

```diff
@@ -1,43 +1,59 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I think these integration test doesn't make much sense. It doesn't really help finding bugs and reviewing the snapshot is nearly impossible to do correctly. Could you remove it? Your snapshot tests are far better.